Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -328,6 +328,24 @@ const ScrollView = React.createClass({
      * @platform android
      */
     scrollPerfTag: PropTypes.string,
+
+     /**
+     * Used to override default value of overScroll mode.
+     *
+     * Possible values:
+     *
+     *  - `'always'` - Always allow a user to over-scroll this view.
+     *  - `'always-if-content-scrolls'` - Default value, allow a user to over-scroll
```

Comment:
I think it would be a nice consistency with `overflow: auto` in css, where the scroll bar only appear in cases where scrolling is possible. It also makes sense because it is the default option.